src/components/FrontPage/TeacherInfobox.js

```diff
@@ -8,15 +8,17 @@ import Collapse from 'react-bootstrap/lib/Collapse';
 import Glyphicon from 'react-bootstrap/lib/Glyphicon';
 import {getTranslator} from '../../selectors/translate';
 
-const TeacherInfobox = React.createClass({
-  getInitialState() {
-    return {
-      showCourseInfo: false
-    };
-  },
+class TeacherInfobox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {showCourseInfo: false};
+    this.changeState = this.changeState.bind(this);
```

Comment:
I stedet for å "bind"-e i constructor, så kan du bare gjøre `changeState` om til en arrow-function. Se f.eks. https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56 (altså, bruk tips nr.5 i stedet for nr.4)